### ISSUE TITLE
ToolTip shows when owner acquires focus via non-keyboard

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4613]
* .NET Core Version: 3.0 and above
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app attached to #4612.
2. Click in the TextBox labeled "TextBox".
3. (optional) wait for ToolTip to appear and disappear (5 sec, 15 sec)
4. Alt+TAB to another app
5. Alt+TAB back to the sample
6. Wait 5 seconds

Steps 4-5 can be replaced by other ways to move focus away from the app, then back to it, e.g.:
4a. Click on app's icon in the system taskbar to minimize the app;  5a. click again to restore the app
4b. Click on another app;  5b. click on sample app's title bar to reactivate it
(etc.)
 
 **Actual behavior:** 
 ToolTip appears

 **Expected behavior:**
ToolTip does not appear
 
ToolTip should display when keyboard navigation (e.g. TAB) moves focus to the TextBox, but not when focus moves by non-keyboard means.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
